.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    env:
+      working_dir: "exercises/webapp"
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+        working-directory: ${{env.working_dir}}
+      - run: npm run lint
+        working-directory: ${{env.working_dir}}
+      - run: npm run test
+        working-directory: ${{env.working_dir}}
```

Comment:
Isn't there a way to [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up? :thinking:

Reply:
Yes its possible. See section jobs.<job_id>.steps.run under https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idsteps